system/preside-objects/taskmanager/taskmanager_task_history_log_line.cfc

```diff
@@ -0,0 +1,18 @@
+/**
+ * Represents a line in a task manager task log
+ *
+ * @versioned      false
+ * @useCache       false
+ * @noLabel        true
+ * @nodatemodified true
+ * @nodatecreated  true
+ */
+component extends="preside.system.base.SystemPresideObject"  {
+	property name="id" required=true type="numeric" dbtype="bigint" generator="increment";
```

Comment:
alignment with "ts" etc

Reply:
as discussed, alignment is within groups, not across groups.